doc/rados/operations/placement-groups.rst

```diff
@@ -747,22 +747,46 @@ To see statistics for a particular PG, run a command of the following form:
 
    ceph pg {pg-id} query
 
-
 Scrub a PG
 ==========
 
-To scrub a PG, run a command of the following form:
+To force an immediate scrub of a PG, run a command of the following form:
+
+.. prompt:: bash #
+
+   ceph tell {pg-id} scrub
+
+or
+
+.. prompt:: bash #
+
+   ceph tell {pg-id} deep-scrub
+
+Ceph checks the primary and replica OSDs, and generates a catalog of all objects in
```

Comment:
s/,//

Reply:
Removed the 1'st ','. 
If you meant s/,//g - let me know...

Reply:
"OSDs and generates" is what I was after.  We documentarians like to be terse.